src/components/Register/Register.js

```diff
@@ -21,11 +23,13 @@ const Register = () => {
     const [name, setName] = useState('');
     const [email, setEmail] = useState('');
     const [password, setPassword] = useState('');
+    const chooseFile = useRef(null);
 
     // submit handler
 
     const handleSubmit = (e) => {
         e.preventDefault();
+        registration.register(name, email, password, chooseFile);
```

Comment:
Don't pass the `chooseFile` ref here - you should have an `onChange` handler on the file input below, and a file state variable that gets updated when a file is chosen. Then, you should pass in the file itself into the `registration.register` function - see more info on how to update the register function below